### PR TITLE
Update OLS-NASA cohorts landing page

### PR DESCRIPTION
**Pull request by iramosp**

### CODE REVIEW

File: nasa/index.md [hunk 1]
```diff
@@ -9,21 +9,40 @@ photos:
...
+|                                 |                                |                       | 1 | 2 | 3 | 4 | 5 | 6 |   
+| **Cohort Calls**                |                                | **Ethos**             | X |   |   |   |   |   |   
+| ^^                              | ^^                             | **Tools & Resources** |   | X |   |   |   |   |  
+| ^^                              | ^^                             | **Data**              |   |   | X |   |   |   |  
+| ^^                              | ^^                             | **Code**              |   |   |   | X |   |   |  
+| ^^                              | ^^                             | **Results**           |   |   |   |   | X |   | 
+| **Final Graduation**            | Group presentations            |                       |   |   |   |   |   | X |
+| **Expert consultation**         | 1 expert:group project         |                       | X | X | X | X | X |   |
+| **Reflection & implementation** | Self-led by participants       |                       | X | X | X | X | X |   \
+|                                 | - optional assignments         |                       |   |   |   |   |   |   \
```
Comment (malvikasharan) [inline nasa/index.md:34]:
```suggestion
|                                 | - Assignments to apply learned skills in research projects  |                       |   |   |   |   |   |   \
```

File: nasa/index.md [hunk 2]
```diff
@@ -9,21 +9,40 @@ photos:
...
+| Cohort timeline                 |                                | TOPS OpenCore modules |  Weeks           ||||||
+|---------------------------------|--------------------------------|-----------------------|:-:|:-:|:-:|:-:|:-:|:-:|
+|                                 |                                |                       | 1 | 2 | 3 | 4 | 5 | 6 |   
+| **Cohort Calls**                |                                | **Ethos**             | X |   |   |   |   |   |   
+| ^^                              | ^^                             | **Tools & Resources** |   | X |   |   |   |   |  
+| ^^                              | ^^                             | **Data**              |   |   | X |   |   |   |  
+| ^^                              | ^^                             | **Code**              |   |   |   | X |   |   |  
+| ^^                              | ^^                             | **Results**           |   |   |   |   | X |   | 
+| **Final Graduation**            | Group presentations            |                       |   |   |   |   |   | X |
+| **Expert consultation**         | 1 expert:group project         |                       | X | X | X | X | X |   |
```
Comment (malvikasharan) [inline nasa/index.md:32]:
```suggestion
| **Expert coaching/consultation**         | 30-45 minutes coaching/consultation with an expert from the program  |                       | X | X | X | X | X |   |
```

File: nasa/index.md [hunk 3]
```diff
@@ -9,21 +9,40 @@ photos:
...
+We will run a pilot cohort in early 2024. 
+
+# Join us
+
+Interested in participating as a trainee or as an expert? Please fill out [this form](https://forms.gle/bUEXdcSkt9BgY4tXA) to register your interest.
 
 # Want more information?
 
-You can read more about it in the [proposal](https://zenodo.org/records/8250979).
+We will be updating this post as we define further details.
```
Comment (malvikasharan) [inline nasa/index.md:46]:
```suggestion
We will be updating this post as we finalize further details. Watch this space!
```

File: nasa/index.md [hunk 4]
```diff
@@ -9,21 +9,40 @@ photos:
...
+|---------------------------------|--------------------------------|-----------------------|:-:|:-:|:-:|:-:|:-:|:-:|
+|                                 |                                |                       | 1 | 2 | 3 | 4 | 5 | 6 |   
+| **Cohort Calls**                |                                | **Ethos**             | X |   |   |   |   |   |   
+| ^^                              | ^^                             | **Tools & Resources** |   | X |   |   |   |   |  
+| ^^                              | ^^                             | **Data**              |   |   | X |   |   |   |  
+| ^^                              | ^^                             | **Code**              |   |   |   | X |   |   |  
+| ^^                              | ^^                             | **Results**           |   |   |   |   | X |   | 
+| **Final Graduation**            | Group presentations            |                       |   |   |   |   |   | X |
+| **Expert consultation**         | 1 expert:group project         |                       | X | X | X | X | X |   |
+| **Reflection & implementation** | Self-led by participants       |                       | X | X | X | X | X |   \
```
Comment (malvikasharan) [inline nasa/index.md:33]:
```suggestion
| **Reflection & implementation** | Self-led by participants - OLS team will provide necessary support  |                       | X | X | X | X | X |   \
```

File: nasa/index.md [hunk 5]
```diff
@@ -9,21 +9,40 @@ photos:
...
+Funded by [NASA](https://www.nasa.gov/), this program will deliver fully virtual training cohorts for the [NASA TOPS OpenCore training curriculum](https://nasa.github.io/Transform-to-Open-Science/) with a 
 highly qualitative component throughout the training. 
 
 # About
       
-Starting in 2024 and for 3 years, the team will deliver five 6-week Virtual Cohorts per year. The cohorts will include guided reflection and hands-on implementation of open science concepts, processes, and 
-techniques. This program will teach one OpenCore Module per week, by bringing in expert speakers from across different scientific domains, allowing learners to actively embed their training in their work and 
-reflect on their learning with a mentor.
+Over three years starting in 2024, we will deliver five 6-week Virtual Cohorts per year. The cohorts will engage participants in hands-on implementation of open science concepts and tools. This program will teach one OpenCore Module per week, by bringing in speakers from across different scientific domains, allowing learners to actively embed their training in their work and reflect on their learning with guidance from expert practitioners.
```
Comment (malvikasharan) [inline nasa/index.md:17]:
```suggestion
OLS cohorts under the NASA-TOPS programme will offer 6-week-long Virtual Cohorts—not one, but five of them over the next three years.

These cohorts will provide structured training, including assignments, enabling learners to actively implement open science concepts and tools in their work. We will delve into one open science module per week, with expert speakers from diverse scientific domains guiding hands-on learning.

Assignments will facilitate reflection on learned concepts, and expert coaching/consultation will provide deeper insights and guidance from practitioners within the context of research.

*Transform your approach to open science dynamically over the next three years with us. Join us and redefine your scientific journey!*
```

File: nasa/index.md [hunk 6]
```diff
@@ -9,21 +9,40 @@ photos:
   url: https://flic.kr/p/2gHMJah
 ---
 
-Funded by [NASA](https://www.nasa.gov/), this work aims to deliver fully virtual training cohorts for the [NASA TOPS OpenCore training curriculum](https://nasa.github.io/Transform-to-Open-Science/) with a 
+Funded by [NASA](https://www.nasa.gov/), this program will deliver fully virtual training cohorts for the [NASA TOPS OpenCore training curriculum](https://nasa.github.io/Transform-to-Open-Science/) with a
```
Comment (malvikasharan) [inline nasa/index.md:12]:
```suggestion
Get ready for an exciting project starting in 2024 in collaboration with !NASA-TOPS](https://nasa.github.io/Transform-to-Open-Science/). 
```

File: nasa/index.md [hunk 7]
```diff
@@ -9,21 +9,40 @@ photos:
   url: https://flic.kr/p/2gHMJah
 ---
 
-Funded by [NASA](https://www.nasa.gov/), this work aims to deliver fully virtual training cohorts for the [NASA TOPS OpenCore training curriculum](https://nasa.github.io/Transform-to-Open-Science/) with a 
+Funded by [NASA](https://www.nasa.gov/), this program will deliver fully virtual training cohorts for the [NASA TOPS OpenCore training curriculum](https://nasa.github.io/Transform-to-Open-Science/) with a 
 highly qualitative component throughout the training.
```
Comment (malvikasharan) [inline nasa/index.md:13]:
```suggestion
Funded by [NASA](https://www.nasa.gov/), OLS will bring its globally successful cohort-based approach, incorporating a personal qualitative component throughout the training.
```

**Review by malvikasharan — APPROVED** [review]
Minor editing suggested to make the text engaging.

File: nasa/index.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Open Science Accessible
+title: OLS & NASA Transform to Open Science
 description: Teaching NASA-TOPS OpenCore in English
```
Comment (malvikasharan) [inline nasa/index.md:4]:
```suggestion
description: Teaching NASA-TOPS Open Science 101 in English
```

File: nasa/index.md [hunk 9]
```diff
@@ -9,21 +9,40 @@ photos:
...
-techniques. This program will teach one OpenCore Module per week, by bringing in expert speakers from across different scientific domains, allowing learners to actively embed their training in their work and 
-reflect on their learning with a mentor.
+Over three years starting in 2024, we will deliver five 6-week Virtual Cohorts per year. The cohorts will engage participants in hands-on implementation of open science concepts and tools. This program will teach one OpenCore Module per week, by bringing in speakers from across different scientific domains, allowing learners to actively embed their training in their work and reflect on their learning with guidance from expert practitioners.
 
-# Expected timeline
+# Timeline
 
-# Interested?
+The training will cover [the five modules in OpenCore](https://opensciency.github.io/sprint-content), and participants will work with an expert to discuss real-world application of the skills they learn. The schedule for individual cohorts is shown below:
```
Comment (malvikasharan) [inline nasa/index.md:21]:
```suggestion
The training will cover the five modules of the [Open Science 101 Curriculum](https://nasa.github.io/Transform-to-Open-Science/), and participants will work with an expert to discuss the real-world application of the skills they learn. The schedule for individual cohorts is shown below:
```

File: nasa/index.md (hunk 6 above)
Comment (iramosp) [reply nasa/index.md:12]:
```suggestion
Get ready for an exciting project starting in 2024 in collaboration with [NASA-TOPS](https://nasa.github.io/Transform-to-Open-Science/).
```

File: nasa/index.md [hunk 10]
```diff
@@ -1,29 +1,88 @@
...
+
+# Program overview
+
+The OLS-NASA Open Science 101 program is a comprehensive 6-week training program designed for participants at any career stage. The program integrates practical training with individual coaching, supporting participants to apply open science principles, skills, and tools in their work.
+
+# Who is this training for?
+
+This training is aimed at researchers and research-supporting staff, from undergraduates to senior scientists. Our program is accessible to participants at any career level, fostering an inclusive environment for those taking their first steps in open science. It will be of particular interest to **individuals or teams** seeking to develop or improve an Open Science strategy or Data Management plan for their research work, whether it is in the proposal stage or already prepared to disseminate results.
+
+Ideas for short projects to work during the program include:
```
Comment (yochannah) [inline nasa/index.md:30]: Love these sample ideas!

**Review by yochannah — APPROVED** [review]

**Comment by yochannah** [comment]
Thanks Irene! I've read through and it all looks like we're ready to merge. Thanks for paying such careful detail.

**Pull request merged**